server/server.go

```diff
@@ -31,6 +31,15 @@ import (
 	"github.com/weaveworks/common/signals"
 )
 
+// Dial connects to the address on the named network
```

Comment:
This change isn't about Dial but Listen.

Reply:
ok ,I fix it